design/src/risk-assessment.md

```diff
@@ -0,0 +1,70 @@
+# Risk Assessment
+
+This page keeps track of risks that we might face during development so that we can mitigate or even eliminate them.
+
+## Organizational Risks
+
+### Feature Creep
+
+[_Feature creep_](https://en.wikipedia.org/wiki/Feature_creep) refers to the continuos addition of features that goes way beyond the initially planned scope of the game.
+We are quite vulnerable to this for multiple reasons:
+
+- The game genre and theme invites for many complex features and mechanics that can be combined freely.
+  It's very easy to come up with additional ideas that would fit well into the game.
+
+- As an open source project, we are going to have many different contributors with different backgrounds.
+  Everyone might have their own ideas that they want to integrate into the game.
+
+To mitigate this risk, the following will be important:
+
+- Clearly document the vision of the game in the design book.
+  This will help to align all contributors on what we want to build.
+
+- Focus on the core game loop first and then iteratively plan the next set of features we want to add.
+  Keeping the scopes small for each iteration will ensure that we don't end up starting too many projects at once.
+
+### Financial Instability
+
+As an open source project it will be harder to generate funding for the work we cannot do ourselves.
+As an indie studio, there is also the uncertainty how popular the game will be in the end.
+Players could also end up compiling the game from source for free without buying the game.
+
+A Patreon model or something similar could help with this.
```

Comment:
Challenge with the Patreon model is pressure to produce (every month) continuous updates. 

Additional possibilities: Github sponsors (especially to get donations from the bevy community)